RELEASE.md

```diff
@@ -1,15 +1,14 @@
 **NOTE:** This guide is only useful for the owners of the venv-update project.
 
-1. `git checkout upstream/master -b {{branch}}`
+1. start on the latest master
 1. bump `venv_update.py`
+1. bump the URL in the `docs/source/index.rst` curl command with the new
```

Comment:
I'd rather have a "latest" tag or "release" branch. The release instructions will be no more annoying, but as a user (or API) I don't need to consult instructions to do the right thing.